pkg/ingester/tenants_http.go

```diff
@@ -19,7 +19,16 @@ import (
 
 type tenantsPageContent struct {
 	Now     time.Time
-	Tenants []string
+	Tenants []tenantsStats
+}
+
+type tenantsStats struct {
```

Comment:
[Nit] I would name it `tenantStats`, since each represents a single tenant, right.

```suggestion
type tenantStats struct {
```

Reply:
Yes, fixed. Thanks.